sdk/python/tests/integration/online_store/test_remote_online_store.py

```diff
@@ -217,6 +218,20 @@ def _start_feature_server(repo_path: str, server_port: int):
         timeout_msg=f"Unable to start the feast server in {_time_out_sec} seconds for remote online store type, port={server_port}",
     )
 
+    if metrics:
+        cmd.append("--metrics")
+
+    logging.debug(f"Starting Feast server with command: {' '.join(cmd)}")
+
+    # Check if metrics are enabled and Prometheus server is running
+    if metrics:
+        logging.debug("Metrics are enabled, checking Prometheus server on port 8000.")
+        wait_retry_backoff(
+            lambda: (None, check_port_open("localhost", 8000)),
+            timeout_secs=_time_out_sec,
+            timeout_msg="Unable to start the Prometheus server in 60 seconds.",
+        )
```

Comment:
can you add an `else` test that ensures its NOT running when metrics are disabled?